Tests/UIKitTests/UIColorExtensionsTests.swift

```diff
@@ -124,6 +124,32 @@ class UIColorExtensionsTests: XCTestCase {
 
     }
 
+    func testCoreImageColor() {
```

Comment:
Doesn't this need to be wrapped as well? I don't think we're running watchOS tests on Travis which allows this to pass.